spaceapi_server/src/datastore/redis_store.rs

```diff
@@ -32,6 +34,10 @@ impl DataStore for RedisStore {
         Ok(try!(con.del(key)))
     }
 
+    fn make_safe(self) -> SafeDataStore {
+        Arc::new(Mutex::new(Box::new(self)))
+    }
+
```

Comment:
Isn't there a generic way to implement `make_safe` for all `DataStore`? I tried something like
```rust
fn make_safe<T: DataStore>(T data_store) -> SafeDataStore {
    Arc::new(Mutex::new(Box::new(self)))
}
```
But I couldn't get it to work...